caravel/assets/spec/javascripts/sqllab/App_spec.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import App from '../../../javascripts/SqlLab/components/App';
+import { describe, it } from 'mocha';
+import { expect } from 'chai';
+
+
+describe('App', () => {
+  const mockedProps = {
+  };
+  it('renders', () => {
+    expect(React.isValidElement(<App />)).to.equal(true);
```

Comment:
this is only testing that it's a valid element, it's not actually rendering, could update the test title to reflect this...